src/me/mrCookieSlime/Slimefun/listeners/BackpackListener.java

```diff
@@ -41,27 +45,41 @@ public void onClose(InventoryCloseEvent e) {
 	}
 	
 	@EventHandler
+	public void onItemDrop(PlayerDropItemEvent e) {
+		ItemStack item = e.getItemDrop().getItemStack();
+		SlimefunItem sfItem = SlimefunItem.getByItem(item);
```

Comment:
Heavy method right there, definitely advised to be moved inside the if-statement